Applications/GesturePod/onComputer/README.md

```diff
@@ -0,0 +1,46 @@
+GesturePod *(Gesture Recognition Pipeline)*
+=========================
+
+## Table of Contents
+
+- [About](#about)
+- [Quick Start](#quick-start)
+- [Docs](#docs)
+- [Also](#also)
+
+## About
+
+GesturePod is a plug-and-play, gesture recognition device. 
+
+As the GesturePod involves hardware, this implements GesturePod's gesture recognition pipeline on your system. We have stored sensor data collected from the GesturePod. This pre-stored sensor data is used for gesture recognition. You can now play around with GesturePod on your system!
+
+## Quick Start
+
+1. Installation requires 
+	- [git](https://git-scm.com/), 
+	- [make](https://www.gnu.org/software/make/)
+	- [g++](https://gcc.gnu.org/).
+
+2. Clone the repo and navigate to this directory
+	```
+	git clone https://github.com/Microsoft/EdgeML
+	cd EdgeML/Applications/GesturePod/onComputer
+	```
+3. Compile and build the code
+	```
+	make
+	```
+4. Run
+	```
+	./gesturepodsim
+	```
+5. By default the program runs for data that has the following gesture: *Double Taps*.
```

Comment:
@ShishirPatil  So, this program is for taps or all gestures? The readme is confusing here. Why do you have a separate file for taps and a common data file?

Reply:
The program runs for the "gesture" contained in the data file present in ```./data/``` directory. Right now, the only gesture contained is Taps. Readme updated to reflect this, and articulate it better.

Reply:
You are assuming a lot of things here - a lot can go wrong. As far as I understand, the point of this code-isolation was to help people who want to understand/play-around with the gesture recognition pipeline. There is no need for us to support arbitrary number of data files. I think one data file is enough. 

Some of the assumptions include:
  1. The `./data/` directory will always exist
  2. All files in that directory will contain your data.
  3. All files will be parseble with your parser. 
  4. You will be able to handle arbitrary time-shifts that will happen between files. 

Any of these will not hold true when people start modifying/playing around with these files.

We can do without these complications.

Reply:
Fixed.